Makefile

```diff
@@ -79,9 +77,9 @@ status: ## Send status request
 send-api-req-local: ## Send a request to the local running server
 	curl -X POST http://localhost:8080/connector -H 'Content-Type: application/json' -H 'Authorization: Bearer dummy-cn389ncoiwuencr' --data-binary "@./fixtures/spotifyPerformance.json" 
 
-.PHONY: send-analytics-req-local
-send-analytics-req-local: ## Send analytics request to the local running server
-	curl http://localhost:8080/analytics/v1/2/episodesAge -H 'Content-Type: application/json' -H 'Authorization: Bearer dummy-cn389ncoiwuencr'
+.PHONY: send-analytics-%
+send-analytics-%: ## Send analytics request to the local running server
+	curl http://localhost:8080/analytics/v1/3/$*?start=2023-01-01\&end=2023-07-01 -H 'Content-Type: application/json' -H 'Authorization: Bearer dummy-cn389ncoiwuencr'
```

Comment:
Maybe we should not hardcode the end date here. Might get outdated pretty quickly. How about 

```bash
start=$(date -d "2 months ago" +%Y-%m-%d)
end=$(date +%Y-%m-%d)
```

Reply:
good idea